doc/install.rst

```diff
@@ -79,19 +77,26 @@ We recommend following the instructions further on to install GMT 6.
 Dependencies
 ------------
 
-PyGMT requires the following libraries:
+PyGMT requires the following libraries to be installed:
 
-* `numpy <http://www.numpy.org/>`__
-* `pandas <https://pandas.pydata.org/>`__
-* `xarray <http://xarray.pydata.org/>`__
-* `netCDF4 <https://github.com/Unidata/netcdf4-python>`__
-* `packaging <https://pypi.org/project/packaging/>`__
+* `numpy <https://numpy.org>`__
+* `pandas <https://pandas.pydata.org>`__
+* `xarray <https://xarray.pydata.org>`__
+* `netCDF4 <https://unidata.github.io/netcdf4-python>`__
+* `packaging <https://packaging.pypa.io>`__
 
 The following are optional (but recommended) dependencies:
 
-* `IPython <https://ipython.org/>`__: For embedding the figures in Jupyter
-  notebooks.
+* `IPython <https://ipython.org>`__: For embedding the figures in Jupyter notebooks.
 
+Note that PyGMT has adopted
+`NEP29 <https://numpy.org/neps/nep-0029-deprecation_policy.html>`__ alongside the rest
+of the Scientific Python ecosystem, and therefore supports:
+
+* All minor versions of Python released 42 months prior to the project,
+  and at minimum the two latest minor versions.
+* All minor versions of NumPy released in the 24 months prior to the project,
+  and at minimum the last three minor versions.
```

Comment:
Is this the best place to put the NEP29 policy? Or should we restructure the Install page a bit more (e.g. put the 'Using Python' (and 'Using GMT'?) section under 'Dependencies' too).

On the first point, `xarray` placed the NEP29 policy language on their install page (see https://xarray.pydata.org/en/v0.17.0/installing.html#minimum-dependency-versions), but not so sure about other projects.

Reply:
NEP29 was proposed [by maintainers of Matplotlib, scikit-learn, IPython, Jupyter, yt, SciPy, NumPy, and scikit-image](https://numpy.org/neps/nep-0029-deprecation_policy.html#abstract), but it seems none of them list the policy on their sites. Perhaps remove the policy and just change `numpy` to `numpy (>=1.17)`, like [matplotlib](https://matplotlib.org/stable/users/installing.html#dependencies).

Instead, we can mention the policy in the maintenance guides.

Reply:
Matplotlib states it at https://matplotlib.org/3.3.4/devel/min_dep_policy.html.

Reply:
Yes, but it's in "Developers' Guide", and most users won't read it.

Reply:
True, I'll move the policy out of the installation page then.